modules/db_instance/outputs.tf

```
@@ -28,6 +28,16 @@ output "db_instance_endpoint" {
   value       = try(aws_db_instance.this[0].endpoint, "")
 }
 
+output "db_instance_engine" {
+  description = "The RDS engine type"
+  value       = try(aws_db_instance.this[0].engine, "")
+}
+
+output "db_instance_engine_version" {
```

we should carry through the actual attribute name like
```suggestion
output "db_instance_engine_version_actual" {
```